### PR TITLE
test(e2e): add e2e tests for shellbar component

### PR DESCRIPTION
**Pull request by Seamoo13**
#### Please provide a link to the associated issue.
Closes SAP/fundamental-ngx#5416

#### Please provide a brief summary of this pull request.
add shellbar core e2e tests and basic visual regression
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/tests/shellbar.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        waitForElDisplayed(shellbarPage.title);
+    }, 1);
+
+    describe('Basic example', function() {
+        it('should open user actions menu on click', () => {
+            scrollIntoView(basicExample);
+            click(basicExample + shellbarAvatar);
+            waitForElDisplayed(popover);
+
+            expect(isElementDisplayed(popover)).toBe(true);
```
Comment (nikvalor) [inline e2e/wdio/core/tests/shellbar.e2e-spec.ts:39]: please add a message if the element is not displayed

File: e2e/wdio/core/tests/shellbar.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            expect(doesItExist(popover)).toBe(false);
+        });
+
+        it('should check user action menu items are clickable', () => {
+            scrollIntoView(basicExample);
+            click(basicExample + shellbarAvatar);
+            const menuItemCount = getElementArrayLength(popoverMenuItem);
+
+            for (let i = 0; i < menuItemCount; i++) {
+                expect(isElementClickable(popoverMenuItem, i)).toBe(true);
```
Comment (nikvalor) [inline e2e/wdio/core/tests/shellbar.e2e-spec.ts:52]: please add a message if the element is not clickable

File: e2e/wdio/core/tests/shellbar.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+            for (let i = 0; i < navItemCount; i++) {
+                expect(isElementClickable(sideNavItems, i)).toBe(true);
+            }
+        });
+
+        it('should check ability to collapse side nav', () => {
+            scrollIntoView(sideNavExample);
+
+            expect(isElementDisplayed(sideNavText)).toBe(true);
```
Comment (nikvalor) [inline e2e/wdio/core/tests/shellbar.e2e-spec.ts:183]: same as above

File: e2e/wdio/core/tests/shellbar.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+            expect(isElementDisplayed(sideNavIcons)).toBe(true);
+
+            click(sideNavControlBtn);
+
+            expect(isElementDisplayed(sideNavText)).toBe(false);
+            expect(isElementDisplayed(sideNavIcons)).toBe(true);
```
Comment (nikvalor) [inline e2e/wdio/core/tests/shellbar.e2e-spec.ts:189]: same as above

File: e2e/wdio/core/tests/shellbar.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+        });
+    });
+
+    describe('shellbar with sidenavigation example', function() {
+        it('should check size nav items clickable', () => {
+            scrollIntoView(sideNavExample);
+            const navItemCount = getElementArrayLength(sideNavItems);
+
+            for (let i = 0; i < navItemCount; i++) {
+                expect(isElementClickable(sideNavItems, i)).toBe(true);
```
Comment (nikvalor) [inline e2e/wdio/core/tests/shellbar.e2e-spec.ts:176]: same as above

**Review by nikvalor — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**